rules/defense_evasion/disable_apparmor_attempt.yaml

```diff
@@ -10,14 +10,50 @@
   type: Exec
   category: defense_evasion
   severity: low
-  description: Monitors for the execution of the systemctl utility with the disable flag to disable the AppArmor service. Attackers may 
-    attempt to disable the AppArmor service to evade detection and avoid possible restrictions on their actions and resources.
-  condition: payload.filename == "/usr/bin/systemctl" AND payload.argv CONTAINS "disable" AND payload.argv CONTAINS "apparmor"
+  description: Monitors for attenot to disable the AppArmor service. Attackers may attempt to disable the AppArmor service 
+    to evade detection and avoid possible restrictions on their actions and resources.
+  condition: (
+    (payload.filename STARTS_WITH "/usr/bin/systemctl" AND payload.argv CONTAINS "disable" AND (payload.argv CONTAINS "apparmor" OR payload.argv CONTAINS "apparmor.service")) OR
+    (payload.filename ENDS_WITH "service" AND payload.argv CONTAINS "apparmor" AND payload.argv CONTAINS "stop") OR
+    (payload.filename ENDS_WITH "chkconfig" AND payload.argv CONTAINS "apparmor" AND payload.argv CONTAINS "off")
+    )
+    
+- name: Move or symlink AppArmor profile to disable
+  type: Exec
+  category: defense_evasion
+  severity: low
+  description: Monitors for the use of ln or mv to relocate AppArmor profiles into the disable directory, which prevents them from being loaded. 
+    Attackers may attempt to disable AppArmor profiles without fully uninstalling them.
+  condition: (payload.filename STARTS_WITH "/usr/bin/ln" AND payload.argv CONTAINS "/etc/apparmor.d/" AND payload.argv CONTAINS "/etc/apparmor.d/disable/")
+    OR (payload.filename STARTS_WITH "/usr/bin/mv" AND payload.argv CONTAINS "/etc/apparmor.d/" AND payload.argv CONTAINS "/etc/apparmor.d/disable/")
```

Comment:
do not rely on args but switch to `FileRename` use `/usr/bin/ln` as image name and put conditions on file path because in file rename events are resolved in absolute form